src/Angor/Client/Pages/View.razor

```diff
@@ -710,16 +702,16 @@
         input = Regex.Replace(input, @"<style.*?>.*?</style>", string.Empty, RegexOptions.IgnoreCase | RegexOptions.Singleline);
 
         input = Regex.Replace(input, @"<([a-zA-Z][^\s>]*)(\s+[^>]*)?>", match =>
-     {
-     string tag = match.Groups[1].Value;
-     string attributes = match.Groups[2].Value;
+        {
+            var tag = match.Groups[1].Value;
```

Comment:
check if the reformat change it to be var instead of a string? i have no recollection of changing this